.golangci.example.yml

```diff
@@ -163,21 +183,12 @@ linters-settings:
       - io.Copy(os.Stdout)
 
   errchkjson:
-    # with check-error-free-encoding set to true, errchkjson does warn about errors
-    # from json encoding functions that are safe to be ignored,
-    # because they are not possible to happen (default false)
-    #
-    # if check-error-free-encoding is set to true and errcheck linter is enabled,
-    # it is recommended to add the following exceptions to prevent from false positives:
-    #
-    #     linters-settings:
-    #       errcheck:
-    #         exclude-functions:
-    #           - encoding/json.Marshal
-    #           - encoding/json.MarshalIndent
-    #           - (*encoding/json.Encoder).Encode
```

Comment:
This has been the result of a lengthy discussion in #2349 and #2362. Me personally would not recommend to remove the section about `linters-settings`, because as mentioned in the above comment, one will have false positives from `errcheck`, if this setting is set to true without the exceptions.

Reply:
@butuzov can you restore this section

Reply:
Clear. thank you, this wasn't clear while cleaning examples.

Reply:
@ldez will do that tomorrow ( not allowed to use git while reading fairy tales for my son)

Reply:
I edited your PR because I want to create another one base on yours.

Reply:
got it.